app/controllers/UserTypeController.scala

```diff
@@ -53,7 +55,11 @@ class UserTypeController @Inject()(identify: IdentifierAction,
     if (cyaMode) {
```

Comment:
I think this needs to be a match on checkMode and updateCaseEnabled because the CYA for Update Case is going to be different from Create case anyway.